.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+          required: false
+        - label: I have checked the **[frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and performed a **[search of the issue tracker (including closed ones)](https://github.com/qbittorrent/qBittorrent/issues)** to avoid posting a duplicate.
+          required: true
+        - label: This is not a support request or question, both of which are better suited for either the **[discussions section](https://github.com/qbittorrent/qBittorrent/discussions)**, **[forum](https://qbforums.shiki.hu/)**, or **[subreddit](https://www.reddit.com/r/qBittorrent/)**. The **[wiki](https://github.com/qbittorrent/qBittorrent/wiki)** did not contain a suitable solution either.
+          required: true
+        - label: I understand that I must provide the qBittorrent preferences file, qBittorrent.log & watched_folders.json (if using "Watched Folders" feature) if relevant to issue/when asked. See **[Where does qBittorrent save its settings?](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings)**
+          required: true
+
+  - type: textarea
+    attributes:
+      label: qBittorrent & operating system versions
+      description: |
+        Qt and libtorrent-rasterbar versions are required when: 1. You are using linux. 2. You are not using an official build downloaded from our website.
+
+        Example of preferred formatting:
+        qBittorrent: 4.3.7 x64
+        Operating system: Windows 10 Pro 21H1/2009 x64
+        Qt: 5.15.2
+        libtorrent-rasterbar: 1.2.14
+      placeholder: |
+        qBittorrent:
+        Operating system:
+        Qt:
+        libtorrent-rasterbar:
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What is the problem?
+      description: Please add the "crash report" (if encountered) or give a clear and concise description of problem.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Please provide reliable steps to reproduce the problem.
+      placeholder: |
+        1. First step
+        2. Second step
+        3. and so on...
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Additional context
+      description: Add screenshots etc. (Anything that will provide more context about the problem)
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Log(s) & preferences file(s)
+      description: |
+        Add these files: qBittorrent preferences file, qBittorrent.log & watched_folders.json (if using "Watched Folders" feature).
+        See **[Where does qBittorrent save its settings?](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings)**
+        #### Note: It's users responsibility to redact any sensitive information
```

Comment:
Very last comment from me (same for the feature request form):
```suggestion
        #### Note: It's the user's responsibility to redact any sensitive information
```

Reply:
>Very last comment from me (same for the feature request form):

It's not in feature request section?!

>#### Note: It's the user's responsibility to redact any sensitive information

will change

Reply:
@thalieht done (bug_report)

Reply:
>It's not in feature request section?!

Oops, don't know what i was looking at, sorry.

Reply:
Just a nitpick (you don't have to do it).

Maybe the 'Extra info/examples/attachments' section in feature_request.yml should also have =

Note: It's the user's responsibility to redact any sensitive information

Reply:
@ArcticGems IMO I don't think it's really required as more than likely it'll be screenshots, links or a wall of text of what the user wants......... but of course that doesn't mean I won't add it or am objecting it, if others also think it's a good idea to do so - it can even be added in the future if really required.